app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OrdersController < ApplicationController
+  skip_before_action :authorize_admin, only: [:show, :new,:create,:add_product, :cart,
+                                              :remove_product, :order_send]
+
+  def new
+  end
+
+  def create
+    @order=current_user.orders.new
+  end
+
+  def index
+    @orders = Order.is_sent_not_processed
```

Comment:
rename this to `not_processed`  Don't use `is_`prefix for methods - read more [here](https://github.com/rubocop-hq/ruby-style-guide#bool-methods-prefix)